### PR TITLE
Manned aviation traffic avoidance (ADS-B, FLARM)

### PR DESCRIPTION
**Pull request by LorenzMeier**
This is a first baseline implementation that allows to respond to aircraft in the vicinity.

### CODE REVIEW

**Comment by tops4u** [comment]
I'm heavily interessted in this and will probably be able to do some Testing with a Beta FLARM Device. Is rhere some more Info about Availability and Config Requirements.

**Comment by LorenzMeier** [comment]
Bringing this in as warning-only, for statistical reasons and first adopters.

**Comment by mrpollo** [comment]
@LorenzMeier can you please make sure the default action here is what we agreed on the dev call?

File: src/modules/navigator/navigator_main.cpp [hunk 1]
```diff
@@ -902,6 +909,139 @@ Navigator::load_fence_from_file(const char *filename)
...
+									     traffic_direction);
+							break;
+						}
+
+					case 2: {
+							mavlink_log_critical(&_mavlink_log_pub, "AVOIDING TRAFFIC %s heading %d, returning home", tr.callsign,
+									     traffic_direction);
+
+							// set the return altitude to minimum
+							_rtl.set_return_alt_min(true);
```
Comment (mrpollo) [inline src/modules/navigator/navigator_main.cpp]: Let's remove the failsafe action until we get more clarity on what we need to do

**Comment by LorenzMeier** [comment]
Replaced with: https://github.com/PX4/Firmware/pull/8214

**Pull request closed without merging**